src/platform/plugins/shared/usage_collection/server/usage_counters/usage_counters_service.test.ts

```diff
@@ -188,51 +189,62 @@ describe('UsageCountersService', () => {
       ]
     `);
   });
-
-  it('retries errors by `retryCount` times before failing to store', async () => {
-    const usageCountersService = new UsageCountersService({
-      logger,
-      retryCount: 1,
-      bufferDurationMs,
-    });
-
-    const mockRepository = coreStart.savedObjects.createInternalRepository();
-    const mockError = new Error('failed.');
-    const mockIncrementCounter = jest.fn().mockImplementation((_, key) => {
-      switch (key) {
-        case 'test-counter:counterA:count:server:20210409':
-          throw mockError;
-        case 'test-counter:counterB:count:server:20210409':
-          return 'pass';
-        default:
-          throw new Error(`unknown key ${key}`);
-      }
-    });
-
-    mockRepository.incrementCounter = mockIncrementCounter;
-
-    coreStart.savedObjects.createInternalRepository.mockReturnValue(mockRepository);
-    const { createUsageCounter } = usageCountersService.setup(coreSetup);
-    jest.useFakeTimers();
-    const usageCounter = createUsageCounter('test-counter');
-
-    usageCountersService.start(coreStart);
-    usageCounter.incrementCounter({ counterName: 'counterA' });
-    usageCounter.incrementCounter({ counterName: 'counterB' });
-    jest.runOnlyPendingTimers();
-
-    // wait for retries to kick in on next scheduler call
-    await tick();
-    // number of incrementCounter calls + number of retries
-    expect(mockIncrementCounter).toBeCalledTimes(2 + 1);
-    expect(logger.debug).toHaveBeenNthCalledWith(1, 'Store counters into savedObjects', {
-      kibana: {
-        usageCounters: {
-          results: [mockError, 'pass'],
+  //  requires extended test runtime because of exponential backoff delay for retries
```

Comment:
For our future selves, to remind us why this specific test is allowed to run longer than 5s.